jobs/forms.py

```diff
@@ -5,10 +5,17 @@
 
 class JobForm(forms.ModelForm):
 
-    company_name = forms.CharField(max_length=150, required=True)
+    company_name = forms.CharField(label='Company',
+                                   max_length=500,
```

Comment:
length should match the model field (although the model field should probably be changed to 255 as a standard)